examples/wasm-shell/components/welcome-message.ts

```diff
@@ -1,63 +1,3 @@
-const welcomeMessage = `Welcome to the wasm terminal example!
-
-Making this super long to test ios Keyboard covering the prompt
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-Okay that's enought space! Welcome to the example :)
-`;
+const welcomeMessage = `Welcome to the wasm terminal example!`;
```

Comment:
We don't need to check for this anymore. So I thought is better to keep it smaller by default.

Reply:
Sounds good! Thank you 😄